eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -45,8 +45,19 @@ jobs:
 
       - template: ../steps/common.yml
 
+      - script: |
+          node eng/tools/generate-dev-builds.js "$(Build.BuildNumber)"
```

Comment:
Don't we need to do an install on this script to ensure all the dependencies are installed? Or is that covered somewhere else?

Reply:
so the only dependency that was required to install with this script was jju library, which is already checked in to the code in master as part of rush tool. And that's why we are referencing that library like this : `../../common/lib/jju/parse`